src/Appium.Net/Appium/AppiumOptions.cs

```diff
@@ -8,24 +9,81 @@ namespace OpenQA.Selenium.Appium
     /// </summary>
     public class AppiumOptions : DriverOptions
     {
+        private const string VendorPrefix = "appium";
+        private const string AutomationNameOption = "appium:automationName";
```

Comment:
Can we use VendorPrefix constant instead of hardcoding `appium` prefix in all constants?

Reply:
I'd imagine, derived classes can override it by supplying their own type safe options, but in this case, windows driver isn't w3c compliant

Reply:
so you mean it is impossible in dotnet to change `private const string AutomationNameOption = "appium:automationName"` to `rivate const string AutomationNameOption = $"{VendorPrefix}:automationName"` ?

Reply:
Oh no, it isn't. Not yet anyway, string interpolation for const is in the upcoming language update I think. But I meant that derived option classes will provide their own type safe options with their vendor prefix set. This AppiumOptions class is a copy of what is in v5 branch, and is subject to multiple changes, especially defining which caps are appium, and stripping out what isn't. For the sake of these final v4 releases, it has no meaning